apps/etterlatte-samordning-vedtak/src/main/kotlin/no/nav/etterlatte/samordning/vedtak/SamordningVedtakService.kt

```diff
@@ -0,0 +1,58 @@
+package no.nav.etterlatte.samordning.vedtak
+
+import no.nav.etterlatte.libs.common.behandling.SakType
+import no.nav.etterlatte.libs.common.vedtak.Utbetalingsperiode
+import no.nav.etterlatte.libs.common.vedtak.VedtakType
+import java.time.LocalDate
+import java.time.YearMonth
+
+class SamordningVedtakService(
+    private val vedtaksvurderingKlient: VedtaksvurderingKlient
+) {
+    suspend fun hentVedtak(
+        vedtakId: Long,
+        accessToken: String
+    ): SamordningVedtakDto {
+        val vedtak = vedtaksvurderingKlient.hentVedtak(vedtakId, accessToken)
+
+        if (vedtak.sak.sakType != SakType.OMSTILLINGSSTOENAD) {
+            throw IllegalArgumentException("Ikke tilgang til vedtak")
+        }
+
+        return with(vedtak) {
+            SamordningVedtakDto(
+                vedtakId = vedtakId,
+                sakstype = "OMS",
+                virkningsdato = virkningstidspunkt.toLocalDate(),
+                opphoersdato = null,
+                type = type.toSamordningsvedtakType(),
+                aarsak = null,
+                anvendtTrygdetid = 40,
```

Comment:
Kan vel hentes fra vedtaket nå?

Reply:
har litt jobb her, da VedtakDto som er i bruk akkurat nå ikke mapper hverken `Beregning` eller `Avkorting`.  Det er kanskje derfor du foreslår å benytte `YtelseMedGrunnlagRoute`?

Reply:
Tar denne biten i ny PR for ikke å gjøre denne for omfattende. Resultatene brukes uansett ikke til noe p.t.